build.gradle

```
@@ -14,7 +14,10 @@ repositories {
 
 dependencies {
 
+    implementation('org.springframework.boot:spring-boot-starter-data-jpa:')
+    implementation'org.projectlombok:lombok'
```

```suggestion
    implementation 'org.projectlombok:lombok'
    annotationProcessor 'org.projectlombok:lombok'
```

Gradle 5.x 이후 버전부터는 annotationProcessor 사용해주셔야 합니다!